### PR TITLE
Ria 3720

### PR DESCRIPTION
**Pull request by mkayad**
### JIRA link (if applicable) ###

 https://github.com/hmcts/ia-ccd-e2e-tests/pull/new/RIA-3720

### Change description ###

see ticket

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

File: e2e/features/remote-hearing-case-list.feature [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    And I should see the text `The hearing notice will be sent to all parties.`
+    And I should see the text `You don't need to do any more on this case.`
+
+    When I click the `Close and Return to case details` button
+    Then I should see an alert confirming the case `has been updated with event: List the case`
+
+    When I click the `Overview` tab
+    Then I should see the `Overview` page
+    And I should see the `Hearing details` field
+    And I should see `Remote hearing` for the `Location` field
```
Comment (bradsorour) [inline e2e/features/remote-hearing-case-list.feature:74]: You need the same check for Location in the `Hearing and appointment` tab.

**Review by bradsorour — COMMENTED** [review]
Look to add a check for **Location** in the `Hearing and appointment` tab too. You test the AC requirement with that check.

File: e2e/features/remote-hearing-case-list.feature (hunk 1 above)
Comment (mkayad) [reply e2e/features/remote-hearing-case-list.feature:74]: i added the verification in the hearing and appointment

**Review by bradsorour — DISMISSED** [review]
LGTM

**Review by bradsorour — APPROVED** [review]
LGTM

**Review by makowalski — APPROVED** [review]
LGTM

**Pull request merged**